test/unit/uk/gov/hmrc/exports/services/mapping/CachingMappingHelperSpec.scala

```diff
@@ -104,6 +104,17 @@ class CachingMappingHelperSpec extends WordSpec with Matchers {
         commodity.classifications shouldBe Seq.empty
       }
 
+      "No commodity code or description provided" in {
+        val exportItem = ExportItem("id", commodityDetails = Some(CommodityDetails(None, None)))
+
+        val commodity = new CachingMappingHelper().commodityFromExportItem(exportItem)
```

Comment:
Here for example - should Commodity be 'None' in this case?

Reply:
Yes, I think it should be :) Thank you!

Reply:
This has been addressed!